From 228ef9f624a18d3c17bdb171fb01dc8b06415ff5 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 18 Jun 2018 15:52:54 +0300
Subject: [PATCH 3983/5242] MLK-18626-1 ARM: dts: imx7d-sdb-epdc: Fix pin
 conflict with can2

commit  341773bd9ff3fcc2f841d30a1d6f6d7b3c5150c1 from
https://source.codeaurora.org/external/imx/linux-imx.git

In upstream the flexcan2 regulator was added with its own pinctrl group
and the EPDC_DATA14 pin is claimed even if the can controller is marked
as disabled.

Mark the regulator itself with status = "disabled" so it won't claim the
pin.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Acked-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb-epdc.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi b/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi
index 604bbbc..f283e4e 100644
--- a/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi
+++ b/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi
@@ -18,6 +18,10 @@
 	status = "disabled";
 };
 
+&reg_can2_3v3 {
+	status = "disabled";
+};
+
 &flexcan2 {
 	status = "disabled";
 };
-- 
1.7.9.5

